packages/shared/src/components/search/SearchPanel/SearchPanel.tsx

```diff
@@ -111,9 +112,11 @@ export const SearchPanel = ({ className }: SearchPanelProps): ReactElement => {
     const keyToIndexModifier: Partial<Record<ArrowKeyEnum, number>> = {
       [ArrowKeyEnum.Up]: -1,
       [ArrowKeyEnum.Down]: 1,
+      [ArrowKeyEnum.Left]: -1,
+      [ArrowKeyEnum.Right]: 1,
```

Comment:
Basically now you can move through search panel with left/right arrow keys. I just enabled it globally in panel because it does not make senese to only have it on tags section because then when you click "right" on last tag you go to nowhere and like this you go to the next section.